ui/app/mirrors/create/cdc/schemabox.tsx

```diff
@@ -136,14 +134,11 @@ const SchemaBox = ({
   const handleSchemaClick = (schemaName: string) => {
     if (!schemaIsExpanded(schemaName)) {
       setExpandedSchemas((curr) => [...curr, schemaName]);
-      if (!schemaLoadedSet.has(schemaName)) {
-        setTablesLoading(true);
-        setSchemaLoadedSet((loaded) => new Set(loaded).add(schemaName));
-        fetchTables(sourcePeer, schemaName, peerType).then((tableRows) => {
-          setRows((value) => [...value, ...tableRows]);
-          setTablesLoading(false);
-        });
-      }
+      setTablesLoading(true);
+      fetchTables(sourcePeer, schemaName, peerType).then((tableRows) => {
+        setRows((value) => _.uniqWith([...value, ...tableRows], _.isEqual));
```

Comment:
Should be straight forward to roll a utility function to achieve this same effect without bringing in lodash dependency. Uniqueness could be keyed on a generated string (maybe join fields with `\n`) avoiding `isEqual` cost. At worst sort & dedupe

Reply:
https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L8441
https://github.com/lodash/lodash/blob/4.17.15/lodash.js#L4249

By passing `_.isEqual` lodash is forced to use `O(N*N)` algo